MLUnits/BTSPrediction/server/prediction_server.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import tflite_runtime.interpreter as tflite
+import argparse
+import time
+import json
+import pika
+
+
+flag = True
+
+class ML_Prediction_Server(object):
+    def __init__(self):
+        self.connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
+        self.channel = self.connection.channel()
+
+        self.channel.exchange_declare(exchange='topic_bts', exchange_type='topic')
+
+        self.result = self.channel.queue_declare(queue='rpc_queue', exclusive=True)
+        self.queue_name = self.result.method.queue
+        self.channel.queue_bind(exchange='topic_bts', queue=self.queue_name, routing_key="request.#")
+
+    def on_request(self, ch, method, props, body):
+        start_time = time.time()
+        predict_value = json.loads(str(body.decode("utf-8")))
```

Comment:
The next version should make sure that parameters will be loaded from configuration files or command line arguments.